src/components/App.js

```diff
@@ -5,14 +5,14 @@ import About from "./About";
 
 // pass this data down as props to the child component(s) that need it!
 import user from "../data/user";
-console.log(user);
+console.log(user.bio);
 
 function App() {
   return (
     <div>
       <NavBar />
-      <Home />
-      <About />
+      <Home name = {user.name} city = {user.city} color = {user.color}/>
+      <About bio = {user.bio} links = {user.links}/>
```

Comment:
For React props, the spacing would be same as HTML attributes:
```suggestion
      <Home name={user.name} city={user.city} color={user.color}/>
      <About bio={user.bio} links={user.links}/>
```